Update generated code (#878)
async-aws/aws: src/Service/CognitoIdentityProvider/CHANGELOG.md

### Added

- AWS api-change: Added regions eu-north-1and eu-west-3.

> - AWS api-change: This release adds ability to configure Cognito User Pools with third party sms and email providers for sending notifications to users.

Reviewer: ```suggestion
- AWS api-change: Improve documentation about configuring Cognito User Pools with third party sms and email providers.
```
